helm/polaris/README.md.gotmpl

```diff
@@ -71,7 +71,7 @@ Simply run the `run.sh` script from the Polaris repo root, making sure to specif
 `--eclipse-link-deps` option:
 
 ```bash
-./run.sh --eclipse-link-deps=org.postgresql:postgresql:42.7.4
+./run.sh
```

Comment:
You need to run 

```shell
 helm-docs --chart-search-root=helm
```

And commit the changes.

Reply:
Done